### PR TITLE
SYNCOPE-1782 migration to AdminLTE 4, Bootstrap 5 and Font Awesome 5

### PR DESCRIPTION
**Pull request by andrea-patricelli**

### CODE REVIEW

File: src/site/xdoc/demo.xml
Comment (ilgrosso) [inline src/site/xdoc/demo.xml:1]: This file is used to build the project website, it has nothing to do with Console or Enduser: are you sure it also needs update?

File: src/site/xdoc/index.xml
Comment (ilgrosso) [inline src/site/xdoc/index.xml:1]: This file is used to build the project website, it has nothing to do with Console or Enduser: are you sure it also needs update?

File: src/site/xdoc/demo.xml
Comment (andrea-patricelli) [reply src/site/xdoc/demo.xml:1]: No need to update, at all. Going to restore.

File: src/site/xdoc/index.xml
Comment (andrea-patricelli) [reply src/site/xdoc/index.xml:1]: No need to update, at all. Going to restore.

**Review by ilgrosso — APPROVED** [review]

**Pull request merged**